src/Tests/Moryx.Tests/Moryx.Tests.csproj

```diff
@@ -14,6 +14,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\..\Moryx.AbstractionLayer\Moryx.AbstractionLayer.csproj" />
```

Comment:
Is that fine or should the test move to `Moryx.AbstractionLayer.Tests` @1nf0rmagician ?

Reply:
Either that or alternatively the test could be limited to `IWorkplanStep` and reflection.